client/verta/verta/_deployment/endpoint.py

```diff
@@ -185,7 +188,44 @@ def _create_stage(self, name="production"):
         return response.json()["id"]
 
     def update_from_config(self, filepath):
-        raise NotImplementedError
+        with open(filepath, 'r') as f:
+            config = f.read()
+
+        is_json = True
+        is_yaml = True
+
+        try:
+            update_dict = json.loads(config)
+        except:
+            is_json = False
+
+        if not is_json:
+            try:
+                update_dict = yaml.safe_load(config)
+            except:
+                is_yaml = False
+
+        if not (is_json or is_yaml):
+            raise ValueError("input must be a json or yaml")
```

Comment:
I would recommend making this a bit cleaner and removing levels of indentation by starting with
```python
update_dict = None
```
instead of
```python
is_json = True
is_yaml = True
```
Then checking if it's still `None` each step of the way.
This would also help prevent a potential undefined variable error

Reply:
Also, I recommend catching specific errors with the `except`s (even though it's seemingly obvious what the error would be, it's still a good practice!)

For `json`, use `ValueError` (`JSONDecodeError` doesn't exist in Py2)
For `yaml`, use `yaml.YAMLError`

Reply:
@convoliution Hmm I actualy don't quite understand what you meant. Is this what you're looking for:
```python
        update_dict = None

        with open(filepath, 'r') as f:
            config = f.read()

        try:
            update_dict = json.loads(config)
        except ValueError:
            try:
                update_dict = yaml.safe_load(config)
            except yaml.YAMLError:
                raise ValueError("input file must be a json or yaml")

        return self._update_from_dict(update_dict)
```

Reply:
@nhatsmrt I just realized my suggestion doesn't actually remove a level of nesting, so disregard that statement 😆 

Something like that. Specifically, I was imagining
```python
        update_dict = None

        with open(filepath, 'r') as f:
            config = f.read()

        try:
            update_dict = json.loads(config)
        except ValueError:
            pass

        if update_dict is None:
            try:
                update_dict = yaml.safe_load(config)
            except yaml.YAMLError:
                pass

        if update_dict is None:
            raise ValueError("input file must be a json or yaml")

        return self._update_from_dict(update_dict)
```

Reply:
Done.